git_intro/git_intro.md

```diff
@@ -2,63 +2,71 @@
 
 author:   Rose Hartman
 email:    hartmanr1@chop.edu
-version: 1.0.5
-module_template_version: 3.0.0
+version: 1.1.0
+current_version_description: Updated with information about GitHub
+module_type: standard
+docs_version: 1.2.0
 language: en
 narrator: UK English Female
 title: Intro to Version Control
+mode: Textbook
 
 comment:  An introduction to what version control systems do and why you might want to use one.
 
 long_description: Version control systems allow you to keep track of the history of changes to a text document (e.g. writing, code, and more). Version control is an increasingly important tool for scientists and scientific writers of all disciplines; it has the potential to make your work more transparent, more reproducible, and more efficient. This module is appropriate for beginners with no previous exposure to version control.
 
-estimated_time: 10 min
+estimated_time_in_minutes:  15
+
+@pre_reqs
+
+None. This lesson is appropriate for beginners with no experience using version control. Experience using word processing software like Microsoft Word, Google Docs, or LibreOffice may be helpful but is not required.
+
+@end
 
 @learning_objectives
 
 After completion of this module, learners will be able to:
 
 - Understand the benefits of an automated version control system
 - Understand the basics of how automated version control systems work
+- Explain how git and GitHub differ
 
 @end
 
-link:  https://chop-dbhi-arcus-education-website-assets.s3.amazonaws.com/css/styles.css
-script: https://kit.fontawesome.com/83b2343bd4.js
+good_first_module: true
+coding_required: false
+sequence_name: git
 
-@version_history
-1.0.3: Fix typos
-@end
--->
-
-# Intro to Version Control
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros.md
+import: https://raw.githubusercontent.com/arcus/education_modules/main/_module_templates/macros_r.md
 
-<div class = "overview">
-
-## Overview
-@comment
+@version_history
 
-**Is this module right for me?** @long_description
+Previous versions:
 
-**Estimated time to completion:** @estimated_time
+- [1.0.5](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/9e3ed69c5c70e4b6e116e2799329029e9542ca98/git_intro/git_intro.md#1): Original version, and then fix typos, update highlight boxes, layout corrections
 
-**Pre-requisites**
+@end
+-->
 
-None. This lesson is appropriate for beginners with no experience using version control. Experience using word processing software like Microsoft Word, Google Docs, or LibreOffice may be helpful but is not required.
+# Intro to Version Control
 
+@overview
 
-**Learning Objectives**
+<div class = "gratitude">
```

Comment:
I added the gratitude box here to bring the attribution up to the front, as we do for the R Basics sequence